pkg/logger/logger.go

```diff
@@ -20,6 +20,17 @@
 
 package logger
 
+// LogLevel describes the log level type a logger can write with
+type LogLevel byte
+
+const (
```

Comment:
I would suggest to have a look at the `iota` style, e.g. https://splice.com/blog/iota-elegant-constants-golang/ and change this `LogLevel` definition here.